ride-share.rb

```diff
@@ -0,0 +1,125 @@
+LIBER = {
+
+  "DR0001" => [
+
+    {
+    date: "02/03/2016",
+    cost: 10,
+    rating: 3
+    },
+
+    {
+    date: "02/03/2016",
+    cost: 30,
+    rating: 4
+    },
+
+    {
+    date: "02/05/2016",
+    cost: 45,
+    rating: 2
+    }
+
+  ],
+
+  "DR0002" => [
+
+    {
+    date: "02/03/2016",
+    cost: 25,
+    rating: 5
+    },
+
+    {
+    date: "02/04/2016",
+    cost: 15,
+    rating: 1
+    },
+
+    {
+    date: "02/05/2016",
+    cost: 35,
+    rating: 3
+    }
+
+  ],
+
+  "DR0003" => [
+
+    {
+    date: "02/04/2016",
+    cost: 5,
+    rating: 5
+    },
+
+    {
+    date: "02/05/2016",
+    cost: 50,
+    rating: 2
+    }
+
+  ],
+
+  "DR0004" => [
+
+    {
+    date: "02/03/2016",
+    cost: 5,
+    rating: 5
+    },
+
+    {
+    date: "02/04/2016",
+    cost: 10,
+    rating: 4
+    },
+
+    {
+    date: "02/05/2016",
+    cost: 20,
+    rating: 5
+    }
+
+  ]
+
+}
+
+def super_sum(hash, array, key)
```

Comment:
Consider adding one line comments to outline your code and explain what each section is doing.